lib/net/make_net_client.ts

```diff
@@ -42,8 +42,8 @@ export const makeNetClient = (cfg: Net.ClientConfig, logger?: Logger): Net.Clien
         }
 
         return data
-      } catch (err) {
-        logger.error(`[client.request] ${_opts.method} to ${_url.toString()} failed: ${err.message}`)
+      } catch (err: any) {
```

Comment:
avoid `any`. if you're getting compilation errors because `err` is `unknown`, opt for type assertions in the catch block:
```ts
catch(err) {
  if (isAxiosError(err)) {
    logger?.error(`[client.request] ${_opts.method} to ${_url.toString()} failed: ${err.message}`)
  } else {
    logger?.error('[client.request]', err)
  }
  throw err
}
```